src/network/networkctl.c

```diff
@@ -840,7 +862,9 @@ static int list_links(int argc, char *argv[], void *userdata) {
                                    TABLE_STRING, operational_state,
                                    TABLE_SET_COLOR, on_color_operational,
                                    TABLE_STRING, setup_state,
-                                   TABLE_SET_COLOR, on_color_setup);
+                                   TABLE_SET_COLOR, on_color_setup,
+                                   TABLE_STRING, online_state ?: "unknown",
+                                   TABLE_SET_COLOR, on_color_online);
```

Comment:
Let's add the new 'online' column in later PR after @craftyguy implemented `--json` support in `networkd-dispatcher`.
Adding `Online` field in `status` command is OK.